.github/workflows/ci-library.yml

```diff
@@ -23,9 +23,10 @@ jobs:
       - name: Check Types
         run: yarn workspace @storyblok/field-plugin check:types
       - name: Lint
-        run: yarn eslint packages/field-plugin
+        run: |
+          yarn eslint packages/field-plugin --ignore-pattern "packages/field-plugin/helpers/"
+          yarn workspaces foreach --include "helper-*" run lint
```

Comment:
Why should they run separately?

Does this not solve the problem:

`helper-vue3/.eslintrc.json`
```json
{
  "root": true,
  ...
}
```

Reply:
It didn't work. As far as I understand, the moment you run `yarn eslint packages/field-plugin`, it loads the root eslint config and apply it to all the files below. That `root: true` prevents your eslint process from "going up". So I had to separate them to make them work. Maybe I should just ignore the helpers in the root eslint config, so that I don't have to explicitly exclude them in the command line?